source/io/module_javaScriptInvoke.js

```diff
@@ -287,6 +294,7 @@
             (returnTypeName === 'boolean') ||
             (returnTypeName === 'string') ||
             (returnTypeName === 'undefined') ||
+			(returnTypeName === 'object') ||
```

Comment:
Since an opaque reference lets anything through this check doesn't make sense. I think the whole isValidJavaScriptReturnType function can be removed because we already validate the return value further down: https://github.com/ni/VireoSDK/blob/7cf54a5755337a71f11497ebb812c59a4cd14f84/source/io/module_javaScriptInvoke.js#L331